ouroboros-consensus/src/Ouroboros/Storage/ImmutableDB/Util.hs

```diff
@@ -237,11 +251,14 @@ indexBackfill (RelativeSlot slot) (RelativeSlot nextExpected) lastOffset =
 
 -- | CBOR-based 'EpochFileParser' that can be used with
 -- 'Ouroboros.Storage.ImmutableDB.Impl.openDB'.
```

Comment:
Let's add a comment here that once the ChainDB is available, this function can be deleted altogether.